src/main/java/com/google/sps/data/DummyDataAccess.java

```diff
@@ -1,10 +1,26 @@
 package com.google.sps.data;
 
+import com.google.appengine.api.datastore.Key;
 import java.util.ArrayList;
 import java.util.Collection;
 
-public class DummyDataAccess {
-  public static Collection<Mentor> getRelatedMentors() {
+public interface DummyDataAccess extends DataAccess {
```

Comment:
DummyDataAccess is not an interface it implements DataAccess and provides fake data.

Reply:
i had it as an interface with default implementations because DummyDataAccess should never be instantiated, so if i made it a class, I'd need to add a private constructor to prevent instantiation. I just thought keeping it as an interface would help keep code clean. (https://stackoverflow.com/a/39491477 is where i got the inspiration for this style)